Dockerfile

```diff
@@ -46,4 +61,7 @@ EXPOSE 5000
 
 RUN ls -lR /opt/nvidia/clara
 
+# Activate the non-root user as the default users for the container.
+USER ${USERNAME}
```

Comment:
do you want to execute as `${USERNAME}` always? Or would it be better to supply the user in the helm chart?